Traceback (most recent call last):
  File "/opt/ros/humble/share/rosidl_generator_cpp/cmake/../../../lib/rosidl_generator_cpp/rosidl_generator_cpp", line 38, in <module>
    sys.exit(main())
  File "/opt/ros/humble/share/rosidl_generator_cpp/cmake/../../../lib/rosidl_generator_cpp/rosidl_generator_cpp", line 34, in main
    generate_cpp(args.generator_arguments_file)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_generator_cpp/__init__.py", line 38, in generate_cpp
gmake[2]: *** Deleting file 'rosidl_generator_cpp/lr_msgs/msg/actuator_command.hpp'
gmake[2]: *** Deleting file 'rosidl_generator_c/lr_msgs/msg/actuator_command.h'
    return generate_files(
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_cmake/__init__.py", line 87, in generate_files
    expand_template(
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_cmake/__init__.py", line 149, in expand_template
    interpreter.string(template_content, template_path, locals=data)
  File "/usr/lib/python3/dist-packages/em.py", line 2391, in string
Traceback (most recent call last):
  File "/opt/ros/humble/share/rosidl_generator_c/cmake/../../../lib/rosidl_generator_c/rosidl_generator_c", line 38, in <module>
    sys.exit(main())
  File "/opt/ros/humble/share/rosidl_generator_c/cmake/../../../lib/rosidl_generator_c/rosidl_generator_c", line 34, in main
    generate_c(args.generator_arguments_file)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_generator_c/__init__.py", line 37, in generate_c
    return generate_files(
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_cmake/__init__.py", line 87, in generate_files
    expand_template(
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_cmake/__init__.py", line 149, in expand_template
    interpreter.string(template_content, template_path, locals=data)
  File "/usr/lib/python3/dist-packages/em.py", line 2391, in string
    self.safe(scanner, True, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2401, in safe
    self.parse(scanner, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2417, in parse
    token = scanner.one()
  File "/usr/lib/python3/dist-packages/em.py", line 1975, in one
    if firsts is None:
KeyboardInterrupt
Traceback (most recent call last):
  File "/root/Lunar_ROADSTER/lr_ws/build/lr_msgs/ament_cmake_python/lr_msgs/setup.py", line 4, in <module>
    setup(
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.10/distutils/core.py", line 134, in setup
    self.safe(scanner, True, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2401, in safe
    self.parse(scanner, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2421, in parse
    token.run(self, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 1547, in run
    self.subrun(info[0][1], interpreter, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 1631, in subrun
    ok = dist.parse_command_line()
  File "/usr/lib/python3.10/distutils/dist.py", line 483, in parse_command_line
    token.run(interpreter, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 1425, in run
    interpreter.execute(self.code, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2596, in execute
    _exec(statements, self.globals, locals)
  File "<string>", line 2, in <module>
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_cmake/__init__.py", line 198, in _expand_template
    interpreter.string(content, str(template_path), kwargs)
  File "/usr/lib/python3/dist-packages/em.py", line 2391, in string
    self.safe(scanner, True, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2401, in safe
    self.parse(scanner, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2421, in parse
    token.run(self, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 1425, in run
    interpreter.execute(self.code, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2596, in execute
    _exec(statements, self.globals, locals)
  File "<string>", line 1, in <module>
KeyboardInterrupt
    args = self._parse_command_opts(parser, args)
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 1048, in _parse_command_opts
    nargs = _Distribution._parse_command_opts(self, parser, args)
  File "/usr/lib/python3.10/distutils/dist.py", line 539, in _parse_command_opts
    cmd_class = self.get_command_class(command)
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 893, in get_command_class
    self.cmdclass[command] = cmdclass = ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2465, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2471, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3/dist-packages/setuptools/command/egg_info.py", line 21, in <module>
    from setuptools.command.sdist import sdist
  File "/usr/lib/python3/dist-packages/setuptools/command/sdist.py", line 8, in <module>
    from .py36compat import sdist_add_defaults
  File "/usr/lib/python3/dist-packages/setuptools/command/py36compat.py", line 7, in <module>
    class sdist_add_defaults:
KeyboardInterrupt
Exception ignored in: <function Interpreter.__del__ at 0x74392551e050>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/em.py", line 2094, in __del__
  File "/usr/lib/python3/dist-packages/em.py", line 2157, in shutdown
  File "/usr/lib/python3/dist-packages/em.py", line 2647, in finalize
  File "/usr/lib/python3/dist-packages/em.py", line 2200, in push
AttributeError: '_io.TextIOWrapper' object has no attribute 'push'
Exception ignored in: <function Interpreter.__del__ at 0x743a48c51e10>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/em.py", line 2094, in __del__
  File "/usr/lib/python3/dist-packages/em.py", line 2157, in shutdown
  File "/usr/lib/python3/dist-packages/em.py", line 2647, in finalize
  File "/usr/lib/python3/dist-packages/em.py", line 2200, in push
AttributeError: '_io.TextIOWrapper' object has no attribute 'push'
gmake[2]: *** [CMakeFiles/lr_msgs__cpp.dir/build.make:203: rosidl_generator_cpp/lr_msgs/msg/actuator_command.hpp] Interrupt
gmake[1]: *** [CMakeFiles/Makefile2:265: CMakeFiles/lr_msgs__cpp.dir/all] Interrupt
gmake[2]: *** [CMakeFiles/lr_msgs__rosidl_generator_c.dir/build.make:139: rosidl_generator_c/lr_msgs/msg/actuator_command.h] Interrupt
gmake[1]: *** [CMakeFiles/Makefile2:212: CMakeFiles/lr_msgs__rosidl_generator_c.dir/all] Interrupt
gmake[2]: *** [CMakeFiles/ament_cmake_python_build_lr_msgs_egg.dir/build.make:70: CMakeFiles/ament_cmake_python_build_lr_msgs_egg] Interrupt
gmake[1]: *** [CMakeFiles/Makefile2:447: CMakeFiles/ament_cmake_python_build_lr_msgs_egg.dir/all] Interrupt
gmake: *** [Makefile:146: all] Interrupt
